src/main/java/com/adobe/cq/testing/junit/rules/Page.java

```diff
@@ -142,7 +146,7 @@ protected CQClient getClient() throws ClientException {
      * @throws ClientException if the content cannot be created
      */
     protected void prepare() throws ClientException {
-        if (!getClient().exists(TEMPLATE_ROOT_PATH)) {
+        if (!prepared || !getClient().exists(TEMPLATE_ROOT_PATH)) {
```

Comment:
Instead of adding the same condition to both `if`s, wouldn't it be easier to check whether `prepared` is `true` at the beginning of `prepare()` and immediately return if this is the case?

Reply:
sure, it's a matter of preference, I guess.